sentence_transformers/sparse_encoder/evaluation/SparseBinaryClassificationEvaluator.py

```diff
@@ -123,7 +124,7 @@ def __init__(
             batch_size=batch_size,
             show_progress_bar=show_progress_bar,
             write_csv=write_csv,
-            truncate_dim=truncate_dim,
+            truncate_dim=None,
```

Comment:
```suggestion
```
I think we can totally ignore it as it already defaults to `None`. Same goes for all of the other Sparse evaluators, too